Editor/CoreAPI/StateManager.cs

```diff
@@ -83,6 +83,7 @@ public string AnywhereFleetId
             {
                 _selectedProfile.AnywhereFleetId = value;
                 SaveProfiles();
+                OnUserProfileUpdated?.Invoke();
```

Comment:
Instead of re-adding this event to AnywhereFleetId, consider making sure OnFleetChanged is subscribed to in the correct components: AnywherePage, ConnectToFleetInput, and RegisterComputeInput are the only components that touch the AnywhereFleetId and/or the dependent property ComputeName.